packages/features/ee/workflows/api/handleSMSResponse.test.ts

```diff
@@ -0,0 +1,10 @@
+import type { NextRequest } from "next/server";
+import { describe } from "vitest";
+
+import handleSMSResponse from "./handleSMSResponse";
+
+describe("handleSMSResponse", () => {
+  it("should handle opt out message", async () => {
+    await handleSMSResponse({} as NextRequest);
```

Comment:
@joeauyeung how is this testing something? Are we just checking that it doesn't fail?